src/share/routes.ts

```diff
@@ -194,7 +202,7 @@ function register(router: Router) {
         }
     }
 
-    router.get("/share/", (req, res, next) => {
+    router.get("/share/", (req, res, _next) => {
```

Comment:
If it's not needed, can't we just remove the `next` argument?

Reply:
yes we can :-)
I even had the same idea, see my commit message: 
https://github.com/TriliumNext/Notes/pull/1384/commits/f25f5774c4e708a18ba1239e04ee08ddaa8277fd

will make adjustments accordingly